src/CMakeLists.txt

```diff
@@ -33,15 +33,20 @@ enable_language(CUDA)
 # Ensure this is not an in-source build
 include(${FLAMEGPU_ROOT}/cmake/OutOfSourceOnly.cmake)
 
-# If FLAMEGPU_SHARE_USAGE_STATISTICS environment variable already exists then the default value can be set to true (as a user has already opted-in by setting it)
-set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT OFF)   # default value is off
+# If FLAMEGPU_SHARE_USAGE_STATISTICS environment variable already exists then the default value can be set to based off this
+set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT ON)   # default value is on
 if (DEFINED ENV{FLAMEGPU_SHARE_USAGE_STATISTICS})
     set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT ON)    # assume on unless meets condition below
     # If user has explicitly set sharing to off then set default value to off
     if("$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT}" STREQUAL "FALSE")
         set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT OFF)
-    endif()
-    if("$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT}" STREQUAL "0")
+    elseif("$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT}" STREQUAL "False")
+        set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT OFF)
+        elseif("$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT}" STREQUAL "OFF")
+        set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT OFF)
+        elseif("$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT}" STREQUAL "Off")
+        set (FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT OFF)
+    elseif("$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT}" STREQUAL "0")
```

Comment:
You should be able to get this behaviour just from one if statment, though `False` and `Off` might not work, but given they are being used for CMake stuff, just using the CMakey options seems fair. You'll have to keep it in the string though. 

`if(NOT "$ENV{FLAMEGPU_SHARE_USAGE_STATISTICS_DEFAULT`)` should trigger if it is set to `0`, `OFF`, `NO`, `FALSE`, `N`, `IGNORE`, `NOTFOUND` or the empty string


https://cmake.org/cmake/help/latest/command/if.html#basic-expressions